Bottom: 7b2bcda83e40414aa38b1d8875a90012db70e708
Top:    1f5ab154e741672b4ee79dbf23a9c915e29ef47f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-10 16:47:19 +0000

w32: TMP Build fixes


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..f6a641c 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -12,6 +9,9 @@ tiff
 freetype
 pixman
 libiconv
+"
+
+BUILD="
 gettext
 glib
 atk
